l10n_it_fatturapa_out/tests/data/IT06363391001_00015.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0"?>
+<ns1:FatturaElettronica xmlns:ns1="http://ivaservizi.agenziaentrate.gov.it/docs/xsd/fatture/v1.2" versione="FPA12">
+  <FatturaElettronicaHeader>
+    <DatiTrasmissione>
+      <IdTrasmittente>
+        <IdPaese>IT</IdPaese>
+        <IdCodice>06363391001</IdCodice>
+      </IdTrasmittente>
+      <ProgressivoInvio>00015</ProgressivoInvio>
+      <FormatoTrasmissione>FPA12</FormatoTrasmissione>
+      <CodiceDestinatario>UFPQ1O</CodiceDestinatario>
+      <ContattiTrasmittente>
+        <Telefono>06543534343</Telefono>
+        <Email>info@yourcompany.example.com</Email>
+      </ContattiTrasmittente>
+    </DatiTrasmissione>
+    <CedentePrestatore>
+      <DatiAnagrafici>
+        <IdFiscaleIVA>
+          <IdPaese>IT</IdPaese>
+          <IdCodice>06363391001</IdCodice>
+        </IdFiscaleIVA>
+        <Anagrafica>
+          <Denominazione>YourCompany</Denominazione>
+        </Anagrafica>
+        <RegimeFiscale>RF01</RegimeFiscale>
+      </DatiAnagrafici>
+      <Sede>
+        <Indirizzo>Via Milano, 1</Indirizzo>
+        <CAP>00100</CAP>
+        <Comune>Roma</Comune>
+        <Provincia>AK</Provincia>
+        <Nazione>IT</Nazione>
+      </Sede>
+      <Contatti>
+        <Telefono>06543534343</Telefono>
+        <Email>info@yourcompany.example.com</Email>
+      </Contatti>
+    </CedentePrestatore>
+    <CessionarioCommittente>
+      <DatiAnagrafici>
+        <IdFiscaleIVA>
+          <IdPaese>IT</IdPaese>
+          <IdCodice>06363391001</IdCodice>
+        </IdFiscaleIVA>
+        <CodiceFiscale>06363391001</CodiceFiscale>
+        <Anagrafica>
+          <Denominazione>Public Administration</Denominazione>
+        </Anagrafica>
+      </DatiAnagrafici>
+      <Sede>
+        <Indirizzo>Via Roma, 1</Indirizzo>
+        <CAP>10100</CAP>
+        <Comune>Torino</Comune>
+        <Provincia>AK</Provincia>
+        <Nazione>IT</Nazione>
+      </Sede>
+    </CessionarioCommittente>
+  </FatturaElettronicaHeader>
+  <FatturaElettronicaBody>
+    <DatiGenerali>
+      <DatiGeneraliDocumento>
+        <TipoDocumento>TD01</TipoDocumento>
+        <Divisa>EUR</Divisa>
+        <Data>2021-12-16</Data>
+        <Numero>INV/2021/12/0001</Numero>
+        <ImportoTotaleDocumento>14.00</ImportoTotaleDocumento>
```

Comment:
Eventuale alternativa: Campo `Divisa` in USD e `ImportoTotaleDocumento` in USD e il resto tutto uguale